tools/CMakeLists.txt

```diff
@@ -26,3 +33,35 @@ if (SEA_DSA_STATIC_EXE)
   set_target_properties (seadsa PROPERTIES LINK_SEARCH_START_STATIC ON)
   set_target_properties (seadsa PROPERTIES LINK_SEARCH_END_STATIC ON)
 endif()
+
+file(
+  MAKE_DIRECTORY
+  ${SPEC_DIR}
+)
+
+add_custom_command(
+  OUTPUT ${SPEC_DIR}/libc.spec.c 
+  DEPENDS libcSpecGen 
+  COMMAND libcSpecGen
+  COMMAND clang-format -i ${SPEC_DIR}/libc.spec.c
+)
+
+add_library(LibcSpecs OBJECT ${SPEC_DIR}/libc.spec.c)
+set_target_properties(LibcSpecs PROPERTIES 
+  COMPILE_FLAGS "-emit-llvm" LINK_FLAGS "-emit-llvm")
+
+ install (FILES  ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/LibcSpecs.dir/specs/libc.spec.c.o
+   DESTINATION lib
+   RENAME libc.spec.bc)
+
+option(SEAHORN_BUILD_32_BIT_SPEC "Build 32bit library specs" OFF)
+if(SEAHORN_BUILD_32_BIT_SPEC) 
+  add_library(LibcSpecs-32 OBJECT ${SPEC_DIR}/libc.spec.c)
+  set_target_properties(LibcSpecs-32 PROPERTIES
+    COMPILE_FLAGS "-emit-llvm -m32" LINK_FLAGS "-emit-llvm -m32")
+
+    install (FILES  ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/LibcSpecs-32.dir/specs/libc.spec.c.o
```

Comment:
file path seems wrong, here and in prior cases. I don't think we can make any assumption about the content of `CMAKE_CURRENT_BINARY_DIR`. CMake is free to rename `CMakeFiles` in the future. Usually `CMAKE_CURRENT_BINARY` dir is sufficient.

The main problem is that this code should be moved into CMakeLists file in `specs` directory